src/solvers/dgsem_tree/container_viscous_3d.jl

```diff
@@ -0,0 +1,75 @@
+mutable struct ViscousContainer3D{uEltype <: Real}
```

Comment:
Am I right to assume that this is essentially the same as for 1D but extended to 2D?

Reply:
Same answer as for 2d